pages/frontendcache/utils.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import requests
+from django.conf import settings
+from wagtail.contrib.wagtailfrontendcache.utils import get_backends
+
+import fastly
+
+logger = logging.getLogger(__name__)
+
+CACHE_FRONTEND_ENABLED = settings.CACHE_FRONTEND_ENABLED
+DEBUG = settings.DEBUG
+
+if CACHE_FRONTEND_ENABLED and not DEBUG:
```

Comment:
I couldn't figure out a good way to do this with nested configs currently implemented in settings.py, but un-nesting them created problems. Feedback welcomed here.